pkg/attestlib/pkix_key.go

```diff
@@ -41,10 +41,10 @@ func parsePkixPrivateKeyPem(privateKey []byte) (interface{}, error) {
 	return key, nil
 }
 
-func generatePkixPublicKeyId(privateKey interface{}) (string, error) {
-	switch privateKey.(type) {
+func generatePkixPublicKeyId(key interface{}) (string, error) {
```

Comment:
nit: Add some internal comments for the expected input of this function

Reply:
done.